pkgs/by-name/sp/speechd/package.nix

```diff
@@ -38,9 +38,11 @@ stdenv.mkDerivation (finalAttrs: {
   pname = "speech-dispatcher";
   version = "0.12.1";
 
-  src = fetchurl {
-    url = "https://github.com/brailcom/speechd/releases/download/${finalAttrs.version}/speech-dispatcher-${finalAttrs.version}.tar.gz";
-    sha256 = "sha256-sUpSONKH0tzOTdQrvWbKZfoijn5oNwgmf3s0A297pLQ=";
+  src = fetchFromGitHub {
```

Comment:
This is already built from source. Generally, I prefer using the official upstream distribution method since that is likely the most tested.